docs/source/conf.py

```diff
@@ -159,24 +161,69 @@ def patch_pygments_to_highlight_jsonschema():
 
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
-html_theme = "sphinx_book_theme"
-
+html_theme = "plone_sphinx_theme"
 html_logo = "_static/logo.svg"
 html_favicon = "_static/favicon.ico"
 
-html_css_files = ["custom.css", ("print.css", {"media": "print"})]
+# html_css_files = ["custom.css", ("print.css", {"media": "print"})]
```

Comment:
If you are not going to use it, it is better to remove this line.

Reply:
Done in https://github.com/plone/plone.restapi/pull/1815/commits/4be1632dc2b86e013153b2dbfcb69b2abfc4e962